src/components/misc/ValidatorList/types.ts

```diff
@@ -1,18 +1,27 @@
 export interface ValidatorRaw {
-    connection: boolean
+    connected: boolean
+    delegationFee: string
+    delegators: Array<Delegators>
     endTime: string
     nodeID: string
+    potentialReward: string
+    rewardOwner: ValidatorRewardOwner //object in console
     stakeAmount: string
     startTime: string
+    txID: string
     uptime: string
-    delegatorWeight: string
-    delegatorCount: string
-    delegationFee: string
+}
+
+export interface Delegators {
+    endTime: string
+    nodeID: string
     potentialReward: string
-    rewardOwner: ValidatorRewardOwner
+    rewardOwner: Object
```

Comment:
In validator row, we have ValidatorRewardOwner as type

Reply:
This is for Delegators